fedbiomed/node/cli.py

```diff
@@ -604,42 +638,57 @@ class NodeCLI(CommonCLI):
     def __init__(self):
         super().__init__()
 
-        self._parser.prog = "fedbiomed_run node"
+        self._parser.prog = "fedbiomed node"
         self.description = f"{__intro__} \nA CLI app for fedbiomed node component."
         # Parent parser for parameters that are common for Node CLI actions
         self.initialize()
 
     def initialize(self):
         """Initializes node module"""
 
-
-        class ConfigNameActionNode(ConfigNameAction):
+        class ComponentDirectoryActionNode(ComponentDirectoryAction):
 
             _this = self
             _component = ComponentType.NODE
 
-            def set_component(self, config_name: str) -> None:
+            def set_component(self, component_dir: str | None = None) -> None:
                 """Create node instance"""
-                config = NodeConfig(name=config_name)
+
+                if component_dir:
+                    component_dir = os.path.abspath(component_dir)
+                    os.environ["FBM_NODE_COMPONENT_ROOT"] = component_dir
+                else:
+                    print("Component is not specified: Using 'fbm-researcher' in current working directory...")
+                    component_dir =  os.path.join(os.getcwd(), 'fbm-node')
+                    os.environ["FBM_NODE_COMPONENT_ROOT"] = component_dir
+
+                config = node_component.initiate(component_dir)
                 self._this.config = config
                 node = Node(config)
 
                 # Set node object to make it accessible
-                setattr(ConfigNameActionNode._this, '_node', node)
+                setattr(ComponentDirectoryActionNode._this, '_node', node)
                 os.environ[f"FEDBIOMED_ACTIVE_{self._component.name}_ID"] = \
                     config.get("default", "id")
 
                 # Set node in all subparsers
-                for _, parser in ConfigNameActionNode._this._arg_parsers.items():
+                for _, parser in ComponentDirectoryActionNode._this._arg_parsers.items():
                     setattr(parser, '_node', node)
 
         super().initialize()
 
         self._parser.add_argument(
-            "--config",
-            "-cf",
+            "--path",
+            "-p",
             nargs="?",
-            action=ConfigNameActionNode,
-            default=DEFAULT_CONFIG_FILE_NAME_NODE,
-            help="Name of the config file that the CLI will be activated for."
-                 f"Default is '{DEFAULT_CONFIG_FILE_NAME_NODE}.")
+            action=ComponentDirectoryActionNode,
+            default="fbm-node",
+            help="Name of the config file that the CLI will be activated for. "
+                "Default is 'config_node.ini'."
+        )
```

Comment:
Unclear: from what i understood, it is not really the config file, but rather the folder containing the node component. Same for `Researcher` 's `--path` help message

Reply:
Hi @ybouilla,
Yes you are right. This help text is not updated.